### PR TITLE
Add login tests

### LINKED ISSUE #13: Test user login flow

### PR DESCRIPTION
**Pull request by assertchris**
Closes #13

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
                             <label for="email" class="col-sm-4 col-form-label text-md-right">{{ __('E-Mail Address') }}</label>
 
                             <div class="col-md-6">
-                                <input id="email" type="email" class="form-control{{ $errors->has('email') ? ' is-invalid' : '' }}" name="email" value="{{ old('email') }}" required autofocus>
+                                <input dusk="email" id="email" type="email" class="form-control{{ $errors->has('email') ? ' is-invalid' : '' }}" name="email" value="{{ old('email') }}" autofocus>
```
Comment (m1guelpf) [inline resources/views/auth/login.blade.php]: Why did you remove required here?

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -29,7 +29,7 @@
                             <label for="password" class="col-md-4 col-form-label text-md-right">{{ __('Password') }}</label>
 
                             <div class="col-md-6">
-                                <input id="password" type="password" class="form-control{{ $errors->has('password') ? ' is-invalid' : '' }}" name="password" required>
+                                <input dusk="password" id="password" type="password" class="form-control{{ $errors->has('password') ? ' is-invalid' : '' }}" name="password">
```
Comment (m1guelpf) [inline resources/views/auth/login.blade.php]: Why did you remove required here?

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (assertchris) [reply resources/views/auth/login.blade.php]: Makes it easier to test validation.

File: resources/views/auth/login.blade.php (hunk 2 above)
Comment (assertchris) [reply resources/views/auth/login.blade.php]: Makes it easier to test validation.
